### DISCUSSION TITLE
cherry-pick

### DISCUSSION THREAD

**Discussion by HyunDel** [guui-programmers/Programmers-Algorithm#20] [Q&A] [answered]
git cherry-pick 에서 충돌 일어났을 때 어떻게 해결해야 되져..???

**Comment by thyoondev** [comment, marked as answer]
First you should undo your cherry-pick, try to run this

```
git cherry-pick --abort
```
Second, try to make cherry-pick, but in this time you get their changes not yours, so make this:
```
git cherry-pick --strategy=recursive -X theirs {Imported_Commit}
```
